roles/virtualbmc/molecule/default/prepare.yml

```diff
@@ -32,10 +32,7 @@
           vms:
             compute:
               amount: 1
-              image_url: "{{ cifmw_discovered_image_url }}"
-              sha256_image_name: "{{ cifmw_discovered_hash }}"
-              image_local_dir: "{{ cifmw_basedir }}/images/"
-              disk_file_name: "centos-stream-9.qcow2"
+              disk_file_name: "blank"
```

Comment:
don't understand the relevance of this bit with respect to the commit message though?

Reply:
it will make molecule faster on that role. Not directly related, but we can still consider it as an optimization